### ISSUE TITLE
Consider refactoring samples

### ISSUE THREAD

**Issue by jeffwilcox** [Azure/azure-webjobs-sdk-samples#6]
Hi,
Have you considered refactoring the repo into multiple sample repos in the `Azure-Samples` org, which is then indexed and appearing up on Azure.com at https://azure.microsoft.com/en-us/documentation/samples/?

@bursteg (GUYB alias) can help share more if you're interested.

**Comment by mathewc** [comment]
Yes, we're aware of the new Azure-Samples. Not sure when we'll get around to doing this, but we're tracking it. We'll leave this issue open as another reminder :)

**Comment by ghost** [comment]
2 years have passed and the annoying reminder is still here :)

**Comment by chrisdunne** [comment]
3 years have passed and the annoying reminder is still here :)

**Comment by mathewc** [comment]
What "annoying reminder" are you referring to? This repo isn't active and likely we won't be continuing work on samples here.

**Comment by nateous** [comment]
Is there a sample app that targets 4.7.2 or Core 2.1? I'm confused if there is a limitation on target .net versions or not. I can't get a simple demo-ware SendGrid example working. I'll write a public repo with sample code, but first I need to know if I'm using the wrong framework version. Most samples I see are for framework 4.5. Some are for Core, but not easily reproduced in existing projects.

Can you easily write WebJobs for App Services in `Framework 4.7.2`? Is there a template to start from?

**Comment by OzBob** [comment]
@nateous Did you get a version working for SendGrid?
I am using .Net 4.7.2. and use v2.3.0 WebJobs, the syntax of JobHost changed in v3 so I will learn what that is and fork this repo and maybe upgrade this demo code.

**Comment by nateous** [comment]
@OzBob I'm no longer working on that project. I still use send grid, but I'm using core 2.0 and I'm not using web jobs now. Having those things work together will help people with legacy code for sure.
